README.md

```diff
@@ -1,16 +1,18 @@
 # ECM Distro Tools
 
-ECM Distro Tools (codename: dirt-weasel) is a collection of programs, scripts, and utilities that provide for easier administration, management, and interaction with the RKE2 and K3s ecosystems.
+ECM Distro Tools is a collection of utilities that provide for easier administration, management, and interaction with the great Rancher ecosystems, including RKE2 and K3s.
 
```

Comment:
It would be nice to include a `## Install` section with a snippet to run the install script, like https://github.com/k3s-io/k3s#quick-start---install-script